src/EditorFeatures/CSharpTest/Wrapping/InitializerExpressionWrappingTests.cs

```diff
@@ -0,0 +1,158 @@
+        public async Task TestNoWrappingSuggestions()
+        {
+            await TestMissingAsync(
+@"class C {
+    void Bar() {
+        var test = new[] [||]{ 1 };
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestWrappingShortInitializerExpression()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[] [||]{ 1, 2 };
+    }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            1,
+            2
+        };
+    }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            1, 2
+        };
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestWrappingLongIntializerExpression()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[] [||]{ ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog"" };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"",
+            ""quick"",
+            ""brown"",
+            ""fox"",
+            ""jumps"",
+            ""over"",
+            ""the"",
+            ""lazy"",
+            ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog""
+        };
+     }
+}"
+);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestShortInitializerExpressionRefactorings()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[]
+        [||]{
+            1,
+            2
+        };
+    }
+}",
+@"class C {
+    void Bar() {
+        var test = new[] { 1, 2 };
+    }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            1, 2
+        };
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsWrapping)]
+        public async Task TestLongIntializerExpressionRefactorings()
+        {
+            await TestAllWrappingCasesAsync(
+@"class C {
+    void Bar() {
+        var test = new[]
+        [||]{
+            ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[]
+        {
+            ""the"",
+            ""quick"",
+            ""brown"",
+            ""fox"",
+            ""jumps"",
+            ""over"",
+            ""the"",
+            ""lazy"",
+            ""dog""
+        };
+     }
+}",
+@"class C {
+    void Bar() {
+        var test = new[] { ""the"", ""quick"", ""brown"", ""fox"", ""jumps"", ""over"", ""the"", ""lazy"", ""dog"" };
```

Comment:
can we have a test that shows wrapping after the line so long.

Reply:
Yea, that makes sense. I will definitely add one

Reply:
I added an additional test to check long initializer with a single line and multi-line. It might be redundant to have both, so I can potentially remove the single line test I originally wrote.